docs/experiments-msmarco-passage-subset.md

```diff
@@ -46,6 +52,24 @@ Next, we extract the contents into `data`.
 unzip data/msmarco_ans_small.zip -d data
 ```
 
+We should have these files in `data/msmarco_ans_small/`
+```
+ls data/msmarco_ans_small -1
+qrels.dev.small.tsv
+queries.dev.small.tsv
+run.dev.small.tsv
+scores
```

Comment:
What is scores?

Reply:
It is the MRR@10 scores, it is a part of `data/msmarco_ans_small.zip`
```
cat data/msmarco_ans_small/scores 
#####################
MRR @10: 0.15906651549508694
QueriesRanked: 105
R @1000: 0.8476190476190476
R @50: 0.5619047619047619
#####################
```

Reply:
I see my bad